Gemfile.lock

```diff
@@ -203,7 +203,7 @@ GEM
       mixlib-log (~> 2.0)
       rack (~> 2.0)
       uuidtools (~> 2.1)
-    cheffish (14.0.1)
+    cheffish (14.0.4)
```

Comment:
Do we want to bring the unrelated cheffish update along in this PR? Or correct #1825 ?

Reply:
We need to update Inspec too, and this fixes the script that updates the dependencies, so I figured we could do it all here rather than waiting 2+ hours for all the extra builds

Reply:
![makes sense](https://user-images.githubusercontent.com/517302/48516591-ba36df80-e831-11e8-8d59-0d2b9d969e7a.gif)